src/styles/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-background-color;
+  box-sizing: border-box;
+
+  &__img {
+    margin-top: 33px;
+    margin-inline: 19px;
+    width: $card-img-width;
+    height: $card-img-height;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    margin-inline: 16px;
+    margin-top: 40px;
+    color: $card-title-color;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-subtitle-color;
+    margin-inline: 16px 84px;
```

Comment:
be consistent with your margins, now you have margin collapse, 10'th point in checklist